parsers/manifest_parser.go

```diff
@@ -384,12 +384,20 @@ func (dm *YAMLParser) ComposeTriggers(manifest *ManifestYAML) ([]*whisk.Trigger,
 		pub := false
 		wsktrigger.Publish = &pub
 
-		keyValArr := make(whisk.KeyValueArr, 0)
+		//print warning information when .Source is not empty
 		if trigger.Source != "" {
+			fmt.Println("WARNING: The 'source' YAML key in trigger entity is deprecated. Please use 'feed' instead as described in specifications.")
```

Comment:
@daisy-ycguo can we please use the new Debug method from misc.go? 

e.g.,:
        ```whisk.Debug(whisk.DbgWarn, warningString)``` ?

and please move the warning string to our translation file: ```wski18n/resources/en_US.all.json```